docs/platform/developing/creating.md

```diff
@@ -27,35 +27,35 @@ Image creation requires a kickstart file that describes how to create an image.
 1. Download the original kickstart file:
 
    ```
-   $ wget <Snapshot_date_URL>/builddata/images/<Repository>/image-configurations/<kickstart_file>
+   $ wget <Snapshot_date_URL>/images/<Repository>/<Image_name>/<kickstart_file>
    ```
 
    - For example:
 
      - Tizen: latest: Unified / standard / `mobile-wayland-armv7l-tm1.ks`
 
        ```
-       $ wget http://download.tizen.org/releases/milestone/tizen/unified/tizen-unified_20210513.3/images/standard/mobile-wayland-armv7l-tm1/tizen-unified_20210513.3_mobile-wayland-armv7l-tm1.ks
+       $ wget http://download.tizen.org/releases/milestone/tizen/unified/tizen-unified_20211014.1/images/standard/mobile-wayland-armv7l-tm1/tizen-unified_20211014.1_mobile-wayland-armv7l-tm1.ks
        ```
 
-2. Modify the original kickstart file to include locally built RPMs into the Tizen image.
+2. Modify the original kickstart file to update the 'baseurl' properties and to include locally built RPMs into the Tizen image.
```

Comment:
I think the word ‘baseurl’ should be under code tags to maintain consistency with the representation of ‘baseurl’ in **Line 64**. Please look into this.  
```suggestion
2. Modify the original kickstart file to update the `baseurl` properties and to include locally built RPMs into the Tizen image.
```